data/DEDIS/projects.yaml

```diff
@@ -571,3 +571,33 @@ projects:
     license: GPL-3.0
     date_added: 2023-07-05
     date_updated: 2023-07-05
```

Comment:
shouldn't these dates be changed as well?

Reply:
Only with the changes from Ahmed.